src/templates/organisms.njk

```diff
@@ -184,20 +188,27 @@
                     <div class="a-switch">
                         <label class="a-switch__label">Hide slideshow overlay</label>
                         <div class="a-switch__toggle">
-                            <input type="checkbox" name="input-slideshow" id="input-slideshow" role="switch" aria-checked="false">
-                            <label for="input-slideshow"><span class="u-screen-reader-only">Toggle overlay</span></label>
+                            <input type="checkbox" name="input-slideshow" id="input-slideshow" role="switch"
+                                   aria-checked="false">
+                            <label for="input-slideshow"><span
+                                class="u-screen-reader-only">Toggle overlay</span></label>
                             <div class="m-overlay">
-                                <label for="input-slideshow"><a class="m-overlay__button"><span class="fa fa-close"><span class="u-screen-reader-only">Close overlay</span></span></a></label>
+                                <label for="input-slideshow"><a class="m-overlay__button"><span
```

Comment:
I suppose you've used an auto formatter for line width but this reduces readability which defeats the purpose of the max length